example/symfony50/composer.json

```
@@ -3,7 +3,7 @@
     "license": "proprietary",
     "minimum-stability": "dev",
     "require": {
-        "php": "^7.2.9",
+        "php": "^8.1.0",
```

I'd suggest using `>=7.2.5` to match Symfony: https://github.com/symfony/symfony/blob/0caf9c3395d079a5360be8409f3c82d3d221e9d8/composer.json#L35

```suggestion
        "php": ">=7.2.5",
```